src/sbt-test/sbt-scalafix/cross-build/build.sbt

```diff
@@ -41,6 +40,8 @@ lazy val scala212 = project
     scalaVersion := Versions.scala212,
     scalafixSettings
   )
+  .settings(scalafixConfigSettings(IntegrationTest): _*)
```

Comment:
`inConfig` seems to return the same type as `scalafixConfigSettings`, is there a reason why you moved down all the settings and used the spread operator? it would be nicer if we could keep them inline (for the diff, but mostly for users!)